app/views/sign_up_sheet/review_bids_others_work.html.erb

```diff
@@ -1,113 +1,103 @@
+  <% topic = SignUpTopic.find(topic_id) %>
+  <% if !topic_id.nil? %>
+    <%= "#{topic.topic_identifier.present? ? "#{topic.topic_identifier}: " : ""}#{topic.topic_name}" %>
+  <%end%>
+  <%if !@assignment.is_anonymous%>
+    &nbsp; (author: <%= participant.fullname%>)
+  <%end%>
+  </td>
 <% end %>
 
-<h2>Reviews for "<%= @assignment.name %>"</h2>
 
-<!-- add headers that provide information regarding review numbers -->
-<% if @assignment.num_reviews_allowed.nil? || @assignment.num_reviews_allowed == -1%>
-    <h4>Your instructor expects you to do <%= @assignment.num_reviews_required %> reviews. You are not allowed to do any extra reviews. </h4>
-<% elsif @assignment.num_reviews_allowed == @assignment.num_reviews_required %>
-    <h4>You should perform exactly <%= @assignment.num_reviews_allowed %> reviews </h4>
-<% else %>
-    <h4>You may perform between <%= @assignment.num_reviews_required %> and <%= @assignment.num_reviews_allowed %> reviews</h4>
+<% def display_assignment_reviews_info(assignment, num_reviews_completed) %>
+  <h2>Reviews for "<%= assignment.name %>"</h2>
+  <!-- add headers that provide information regarding review numbers -->
+  <% if assignment.num_reviews_allowed.nil? || assignment.num_reviews_allowed == -1 %>
+    <h4>Your instructor expects you to do <%= assignment.num_reviews_required %> reviews. You are not allowed to do any extra reviews. </h4>
+  <% elsif assignment.num_reviews_allowed == assignment.num_reviews_required %>
+    <h4>You should perform exactly <%= assignment.num_reviews_allowed %> reviews </h4>
+  <% else %>
+    <h4>You may perform between <%= assignment.num_reviews_required %> and <%= assignment.num_reviews_allowed %> reviews</h4>
+  <% end %>
+  <h4>You are required to do <%= assignment.num_reviews_required %> reviews</h4>
+  <h4>Number of reviews left: <%= assignment.num_reviews_allowed - num_reviews_completed %></h4>
 <% end %>
-<h4>You are required to do <%= @assignment.num_reviews_required %> reviews</h4>
-<h4> Number of reviews left: <%= @assignment.num_reviews_allowed - @num_reviews_completed %></h4>
 
-<!-- table with reviews available to complete -->
+
+######################################################################################################################################
+
+#This methods displays flash
+<% = flash_message %>
+  
+# This method displays the number of reviews to be done defined in review_bids_helper
+<%= display_assignment_reviews_info(@assignment, @num_reviews_completed) %>
+
+# -- table with reviews available to complete -->
 <table>
   <% review_no = 1 %>
   <% title = 'Review' %>
-  <!-- loop through each review map -->
+
   <% @review_mappings.each do |map| %>
-      <%@sorted_responses=[]%>
-      <% team = AssignmentTeam.find(map.reviewee_id) %>
-      <% participant = team.participants.first %>
+     # Create sorted_response array to store information from particular map id 
```

Comment:
I don't understand the comment.  What kind of information?  The map id would identify who is reviewing whom, but I'd like a more detailed explanation.